resources/code-renderer.css

```diff
@@ -21,11 +21,25 @@ pre[class*="language-"] {
 
 .line-numbers .line-numbers-rows {
 	background-color: #f0f0f0;
-	margin: -1em 0;
+	margin: -1.15em 0;
```

Comment:
What is this change for? If we're only hiding rows then I wouldn't expect this.

Reply:
While I was working on the task at hand, I noticed that the lines element was not properly aligned with the background div, notice how it is slightly below in the following picture:
![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/a9c7874b-5888-4417-a9ec-116dc8d2f11c)

After the fix:
![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/7de035f8-71c6-4baf-b179-728156ede08e)

Since, it started bothering me visually, so, I fixed it in the CSS, but this should've been committed in a separate request not on this ticket. I have reverted this, will do it on a separate request.

Reply:
We can also notice how the number also looks a bit lower than the code in the previous sample, it will be properly aligned after this change.

Reply:
Reverted